### PR TITLE
Issue #11450 - Email clarification for new version

### LINKED ISSUE #11450: Clarify that emails about new versions are sent to Super Users
Recently we got a few emails like this:

> for some reason I'm regularly getting mails whenever a new Piwik version is being released. Since there's no unsubscribe option in these mails I'd highly appreaciate if you'd remove me from the mailing list.

To make it more clear I would suggest that in our emails about new versions, we add a note such as "You receive this email because you are a Super User on the Piwik at piwik.example.org" - any better suggestion welcome!

### PR DESCRIPTION
**Pull request by fbrzozowski**
That should resolve #11450 
Add clarification in emails about new version for super users. Unfortunately I don't know all languages so it needs to be translated.

### CODE REVIEW

**Comment by sgiehl** [comment]
Thanks for the PR.
Please do not change any language files other than en.json. this will be automatically done later

**Comment by fbrzozowski** [comment]
@sgiehl Sorry this is my first time - didn't know that. Thanks for info/advice. :)

**Comment by mattab** [comment]
Thanks and congrats @fbrzozowski on your first PR :+1: 

Here is my feedback:

* remove the languages changes except in en.json (our translators [use transifex](https://www.transifex.com/projects/p/piwik/) to translate PIwik from English and we have automated commands to fetch translations from there)
* in the english string, change it to `[...] on the Piwik at: %s` and then in the code, replace the placeholder by the value. This is better than appending the value because in some languages they want the value in the middle or some other places and it couldn't be done if we manually put it at the end.

File: plugins/CoreUpdater/UpdateCommunication.php [hunk 1]
```diff
@@ -79,6 +79,8 @@ protected function sendNotifications()
             $message .= "\n\n";
         }
 
+        $message .= Piwik::translate('CoreUpdater_ReceiveEmailBecauseIsSuperUser', $host); 
+        $message .= "\n";
```
Comment (mattab) [inline plugins/CoreUpdater/UpdateCommunication.php]: Change this to `\n\n` to fix the test

**Comment by mattab** [comment]
Changes look good, and now: there are now a couple integration tests failing:

```
1) Piwik\Plugins\CoreUpdater\Test\UpdateCommunicationTest::test_sendNotifications_shouldSentCorrectEmail

Expectation failed for method name is equal to <string:sendEmailNotification> when invoked 1 time(s)

Parameter 1 for invocation Piwik\Plugins\CoreUpdater\UpdateCommunication::sendEmailNotification('CoreUpdater_NotificationSubje...Update', 'ScheduledReports_EmailHello\n\n...ntact/') does not match expected value.

Failed asserting that two strings are equal.

--- Expected

+++ Actual

@@ @@

 CoreUpdater_ReceiveEmailBecauseIsSuperUser

-

 CoreUpdater_FeedbackRequest

 http://piwik.org/contact/'

/home/travis/build/piwik/piwik/plugins/CoreUpdater/UpdateCommunication.php:88

/home/travis/build/piwik/piwik/plugins/CoreUpdater/UpdateCommunication.php:55

/home/travis/build/piwik/piwik/plugins/CoreUpdater/Test/Integration/UpdateCommunicationTest.php:118

/home/travis/build/piwik/piwik/plugins/CoreUpdater/Test/Integration/UpdateCommunicationTest.php:87

2) Piwik\Plugins\CoreUpdater\Test\UpdateCommunicationTest::test_sendNotifications_shouldNotIncludeChangelogIfNotMajorVersionUpdate

Expectation failed for method name is equal to <string:sendEmailNotification> when invoked 1 time(s)

Parameter 1 for invocation Piwik\Plugins\CoreUpdater\UpdateCommunication::sendEmailNotification('CoreUpdater_NotificationSubje...Update', 'ScheduledReports_EmailHello\n\n...ntact/') does not match expected value.

Failed asserting that two strings are equal.

--- Expected

+++ Actual

@@ @@

 CoreUpdater_ReceiveEmailBecauseIsSuperUser

-

 CoreUpdater_FeedbackRequest

 http://piwik.org/contact/'
```

**Comment by mattab** [comment]
Well done @fbrzozowski for your first PR. We're looking forward to your next ones :tada:

**Pull request merged**